examples/interactive_play.py

```diff
@@ -252,6 +262,42 @@ def get_input_vel_ctlr(
             logger.info("[play.py]: Snapping")
             magic_grasp = 1
 
+    if use_joint_control:
+        # Add random noise to human arms but keep global transform
+        (
+            new_joints,
+            root_trans,
+        ) = env._sim.articulated_agent.get_joint_transform()
+        num_joints = len(new_joints) // 4
+        root_trans = np.array(root_trans)
+        new_joints_quat = [
+            mn.Quaternion(
+                mn.Vector3(new_joints[(4 * index) : (4 * index + 3)]),
+                new_joints[4 * index + 3],
+            )
+            for index in range(num_joints)
+        ]
+        rotated_joints_quat = []
+        for index, joint_quat in enumerate(new_joints_quat):
+            random_vec = np.random.rand(3)
+            random_angle = np.random.rand() * 360 / 60
+            rotation_quat = mn.Quaternion.rotation(
+                mn.Rad(random_angle), mn.Vector3(random_vec).normalized()
+            )
+            new_quat = joint_quat
+            if index > 10:
```

Comment:
why only for index > 10?

Reply:
Added a comment to explain. Its just to make the demo look nicer, no particular reason.